src/PeopleSDKAdapter.js

```diff
@@ -0,0 +1,119 @@
+import {Observable, from, fromEvent, merge} from 'rxjs';
+import {PeopleAdapter, PersonStatus} from '@webex/component-adapter-interfaces';
+import {publish, refCount, flatMap, map, filter} from 'rxjs/operators';
+
+const USER_PRESENCE_UPDATE_EVENT = 'event:apheleia.subscription_update';
+
+/**
+ * The `PeopleSDKAdapter` is an implementation of the `PeopleAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to fetch data about a person.
+ *
+ * @export
+ * @class PeopleSDKAdapter
+ * @extends {PeopleAdapter}
+ */
+export default class PeopleSDKAdapter extends PeopleAdapter {
+  constructor(datasource) {
+    super(datasource);
+
+    this.getPersonObservables = {};
+  }
+
+  /**
+   * Returns a proper Enum key for a given PersonStatus value
+   *
+   * @param {string} status
```

Comment:
Can you add the private tag to indicate this is intended to be a helper method? See [@private](https://jsdoc.app/tags-private.html)
```suggestion
   * @private
   * @param {string} status
```